package.json

```diff
@@ -5,7 +5,7 @@
     "url": "https://github.com/ggoodman",
     "twitter": "filearts"
   },
-  "version": "8.1.1",
+  "version": "8.1.2",
```

Comment:
I think we should make this a minor because we introduce the feature to support url-based middleware.